google/ggrc-core
# GGRC-4999 Implement filtering by Task State on Active Cycle tab

# Issue description

Implement filtering by Task State on Active Cycle tab

# Steps to test the changes

- Create Workflow with tasks, 
- Activate it
- Open Active Cycles tab
- Expand tree
- Start Task
- Type in search field: `"Task state" = "In progress"` and Search
Task should be filtered.

# Solution description

Reverted changes from GGRC-4960 with few changes. Added task status indexation. 

# Sanity checklist




# PR Review checklist
